core/discovery/brokerdiscovery/repository_test.go

```diff
@@ -18,27 +18,53 @@
 package brokerdiscovery
 
 import (
+	"encoding/json"
 	"testing"
 	"time"
 
 	"github.com/mysteriumnetwork/node/communication/nats"
 	"github.com/mysteriumnetwork/node/eventbus"
 	"github.com/mysteriumnetwork/node/market"
+	"github.com/mysteriumnetwork/node/money"
 	"github.com/stretchr/testify/assert"
 )
 
+func init() {
```

Comment:
I don't see a point to mock special payment method in this testcase, because production code should handle unserialisable payment method already

Reply:
I suggest to try my branch with you suggested changes.

Reply:
ok, looks still needed to test successful proposal add